tests/scripts/pulp_rpm/test_rpm_sync_publish.sh

```diff
@@ -127,18 +127,18 @@ then
   expect_fail pulp rpm repository sync --repository "cli_test_rpm_repository" --sync-policy foobar
 fi
 
-if pulp debug has-plugin --name "rpm" --min-version "3.25.0.dev"
+if pulp debug has-plugin --name "rpm" --max-version "3.24.0"
```

Comment:
```suggestion
if pulp debug has-plugin --name "rpm" --specifier "<3.25.0"
```
Please check the version again...

Reply:
I'm aware, but I keep hitting the same error unexpectedly (not the same as the sporadic one mentioned above), so I was trying a few different combinations

Reply:
https://github.com/pulp/pulp-cli/actions/runs/7641882905/job/20820247714?pr=884#step:7:148
The nightly container is reporting to run `"rpm": "3.25.0.dev"` so we need to wait until that nightly container built with "3.26.0.dev" I would say.